samples/samples-java/src/main/java/com/function/AddProduct.java

```diff
@@ -0,0 +1,41 @@
+package com.function;
+
+import com.microsoft.azure.functions.HttpMethod;
+import com.microsoft.azure.functions.HttpRequestMessage;
+import com.microsoft.azure.functions.HttpResponseMessage;
+import com.microsoft.azure.functions.HttpStatus;
+import com.microsoft.azure.functions.OutputBinding;
+import com.microsoft.azure.functions.annotation.AuthorizationLevel;
+import com.microsoft.azure.functions.annotation.FunctionName;
+import com.microsoft.azure.functions.annotation.HttpTrigger;
+import com.microsoft.azure.functions.sql.annotation.SQLOutput;
+import com.fasterxml.jackson.core.JsonParseException;
+import com.fasterxml.jackson.databind.JsonMappingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.function.Common.Product;
+
+import java.io.IOException;
+import java.util.Optional;
+
+public class AddProduct {
+    @FunctionName("AddProduct")
+    public HttpResponseMessage run(
+            @HttpTrigger(
+                name = "req",
+                methods = {HttpMethod.POST},
+                authLevel = AuthorizationLevel.ANONYMOUS,
+                route = "addproduct")
+                HttpRequestMessage<Optional<String>> request,
+            @SQLOutput(
+                commandText = "Products",
+                connectionStringSetting = "SqlConnectionString")
+                OutputBinding<Product> product) throws JsonParseException, JsonMappingException, IOException {
```

Comment:
Out of curiosity where did you get this list of exceptions? I have to imagine that isn't all of them - for example if there's a SqlException in the function itself what kind of exception is thrown in the worker?

Not a huge deal so we can leave it like this for now - just want to know what to expect here.

Reply:
The exceptions come from mapper.readValue() in line 36. If I don't add the throws, there are build errors. Ex: `Unhandled exception type JsonProcessingException`